flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkSourceConfig.java

```diff
@@ -24,20 +24,21 @@
 import java.util.List;
 import org.apache.flink.types.Row;
 import org.apache.iceberg.flink.FlinkReadOptions;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.TestTemplate;
 
-public class TestFlinkSourceConfig extends TestFlinkTableSource {
+public class TestFlinkSourceConfig extends TableSourceTestBase {
```

Comment:
Extracted common code from `TestFlinkTableSource` to an abstract base class `TableSourceTestBase`. otherwise, `TestFlinkSourceConfig` also include every test from `TestFlinkTableSource`.